docs/inputs/table-group.md

```diff
@@ -17,26 +17,106 @@ Similar to an [**Input Group**](input-group), a **Table Group** allows the _**de
+The main differences between a **Table Group** and an [**Input Group**](input-group) is:
 
 - The inputs share the same fixed width and cannot be resized.
 - The Python variable available in the **Code** section is a two dimensional nested array (eg. `[[1,2],[3,4]]`).
-- The number of columns and the variable name are specified by the _**developer**_ at the top of the group.
-- There is no restriction on the type of input in each cell.
+- The number of columns and the variable name are specified by the _**developer**_ in the edit table popup.
+
+There are also many similarities including:
+
+- No restrictions on the type of input in each cell.
 - Cells can be left empty.
 
-For example, the following **Table Group** has three columns and two rows. The cell in the second row and second column is left blank. A slider input type is chosen for the input cell in the first row and second column while a single select input type is chosen for the input cell in the second row and third column. The array name available for the **Code** step is `arrayname`.
+For example, the following **Table Group** has three columns and 1 row. The header can be made using RichText input type and does NOT count as a row. The array name available for the **Code** step is `table`.
 
 <div style={{textAlign: 'center'}}>
 
-![img alt](/docs/inputs/table_group.png)
+![img alt](/docs/inputs/table_group_numbered.png)
 
 </div>
 
+1. Drag the **Table Group** to order it vertically amongst the other groups
+2. Edit the **Table Group** properties (described below)
+3. Delete the **Table Group**
+4. Duplicate the last row in the current **Table Group** (similar to duplicate input but for the whole row)
+
+### Editing a table group
+
+To edit the table group, click the :pencil2: icon near the top right side of the table group. A popup will appear with the following options:
+
+<div style={{textAlign: 'center'}}>
+
+![img alt](/docs/inputs/edit_table_group_dialog.png)
+
+</div>
+
+1. **Title** -> the title of the table group
+2. **Name** -> this is the variable name of the table which will be used in the python code
+3. **Columns** -> the number of columns in the table
+
+| :trophy: The following are [premium](https://mecsimcalc.com/pricing) features |
+| ----------------------------------------------------------------------------- |
+
+4. **Dynamic Rows** -> check this box if you want the user to be able to add and delete rows from the table  
+    After checking this box a list of options will appear:  
+   a) **_Max Rows_** -> The maximum number of rows that can be in the table (**NOT including the header row**)  
+   b) **_Min Rows_** -> The minimum number of rows that can be in the table
+   (**NOT including the header row**)  
+   c) **_Row Group Size_** -> The number of rows that are 'combined' together when the user 'Adds' or 'Deletes' rows. (_This should almost always be set to 1 ...unless your row is comprised of multiple lines_)
```

Comment:
This explanation is confusing. Maybe "the number of lines that make a single row that can be added and deleted"??